bundles/auth/org.eclipse.smarthome.auth.oauth2client/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/org.eclipse.smarthome.auth.oauth2.client.OAuthClientService.xml,\
```

Comment:
You can only add the folder

Reply:
Sure?

Reply:
Yes. It is similar to the META-INF/ entry and the MANIFEST.MF is added to the jar. Similarly if we add only OSGI-INF/, the XML files inside it will be added to the JAR.